plugins/workspace-backpack/src/msg.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Translatable messages used in backpack.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+import * as Blockly from 'blockly/core';
+
+/** @type {string} */
+// context menu - Copy all Blocks on the workspace to the backpack.
+Blockly.Msg['COPY_ALL_TO_BACKPACK'] = 'Copy All Blocks to Backpack';
```

Comment:
This is a minor note, but to be more like core blockly it would say "Copy 42 blocks to backpack" (e.g. the delete all blocks option). That would also give more flexibility to language translations by using the built-in tools we have for message interpolation (e.g. if the number shouldn't go at the end, in some language)

Also, I think it should be consistent between "add" and "copy" to backpack

Reply:
I'm not sure that the number of Blocks is as useful of information for a Backpack. The number of Blocks and the number of "items" added to the Backpack is not equivalent.

I have updated "Add" to "Copy" for consistency now.

Reply:
Also, to clarify, the number hat appears to the right of the context menu option in the gif is the count of the current number of items in the Backpack.

Reply:
Sounds good.